locale/Makefile

```diff
@@ -6,19 +6,19 @@
 # make tx-update - download and merge translations from Transifex
 # make clean - clean everything
 #
-DOMAIN = foreman_discovery
-VERSION = $(shell git describe --abbrev=0 --tags)
+DOMAIN = $(shell ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("../*.gemspec")[0]).name')
+VERSION = $(shell ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("../*.gemspec")[0]).version')
 POTFILE = $(DOMAIN).pot
 MOFILE = $(DOMAIN).mo
 POFILES = $(shell find . -name '$(DOMAIN).po')
 MOFILES = $(patsubst %.po,%.mo,$(POFILES))
 POXFILES = $(patsubst %.po,%.pox,$(POFILES))
 EDITFILES = $(patsubst %.po,%.edit.po,$(POFILES))
+JSFILES = $(shell find ../app/assets/javascripts/*/locale -name '$(DOMAIN).js')
 
 %.mo: %.po
 	mkdir -p $(shell dirname $@)/LC_MESSAGES
 	msgfmt -o $(shell dirname $@)/LC_MESSAGES/$(MOFILE) $<
-	touch $(shell dirname $@)/LC_MESSAGES/$(MOFILE)
```

Comment:
I've seen that this can be useful. msgfmt doesn't change the file (and keep the timestamp if the content is the same). That means the .po file can be newer than the .mo file.

Can be a good change to https://github.com/theforeman/foreman_plugin_template/blob/master/locale/Makefile and let other repos also benefit from it.